### PR TITLE
Questionnaire

### PR DESCRIPTION
**Pull request by Marie-Garde**

### CODE REVIEW

**Review by Thoms-C — DISMISSED** [review]
i approve this PR !

File: src/App.js [hunk 1]
```diff
@@ -17,6 +18,7 @@ export default function App() {
         <Route exact path="/" component={Mainpage} />
         <Route path="/agissons" component={Agissons} />
         <Route path="/connexion" component={Connexion} />
+    <Route path="/inscription" component={Inscription} />
```
Comment (LoicBrassart) [inline src/App.js]: **Critical**
Indentation failure -> Netlify was not used -> PR was not read :cry:

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Nope.

**Review by Thoms-C — APPROVED** [review]
Indentation done, nice !

File: src/styled-components/Inscription.jsx [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+
+export const DivLabel = styled.div`
+  display: flex;
+  justify-content: start;
+  flex-direction: column;
+  margin-bottom: 6vh;
+  width: 100%;
+  height: 2.7vh;
+  @media ${Device.mobile} {
+  }
```
Comment (LoicBrassart) [inline src/styled-components/Inscription.jsx:63]: *Advice*
That's not a good idea to add empty blocks.

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**